### DISCUSSION TITLE
Provide styles for readonly input state

### DISCUSSION THREAD

**Discussion by thesobercoder** [saadeghi/daisyui#2819] [Ideas / feature requests]
When using daisyUI with react-hook-form, one thing I've come across is that react-hook-form recommends using the `readonly` attribute instead of the `disabled` attribute, and as such, daisyUI inputs don't have styles declared for the `readonly` state.

**Comment by TeaDrinkingProgrammer** [comment]
+1. For now, I have just copied the disabled styles and changed the modifier, but it would nice if it would be included.
